test/modules/testModule/Module.js

```diff
@@ -1,14 +1,27 @@
-module.exports = function testModule(options, UI) {
+module.exports = function Dynamic(options, UI, util) {
+  // Tests for functions that are available inside modules only
 
   var output;
+
+  // This function is called on every draw.
   function draw(input, callback) {
 
-    var output = function(input) {
-      return input;
+    var step = this;
+
+    this.getPreviousStep();
+    this.getNextStep();
+    this.getInput(1);
+    this.getOutput(1);
+    this.getOptions();
+    this.getFormat();
```

Comment:
Can we assert anything with these?

Reply:
@jywarren I thought we just wanted to check if they throw an error or not, actually som of these like height I can hardcode at best, will that be ok?